roles/kubernetes/preinstall/handlers/main.yml

```diff
@@ -32,4 +36,35 @@
   service:
     name: kubelet
     state: restarted
+  notify:
+    - Preinstall | kube-controller configured
+    - Preinstall | remove controller
+    - Preinstall | pause for controller
+    - Preinstall | restart controller
   when: "{{ inventory_hostname in groups['kube-master'] and not dns_early|bool }}"
+
+#
+# The following is used to restart the kube-controller to pick up
```

Comment:
Note: We've been leaving most of the commentary in docs or in commit messages, trying to avoid inserting a lot of commentary in playbook files themselves.